src/index.tsx

```diff
@@ -448,3 +449,65 @@ const useReactRouterBreadcrumbs = (
 });
 
 export default useReactRouterBreadcrumbs;
+
+// https://github.com/remix-run/react-router/blob/main/packages/react-router/index.tsx#L760
+// The createRoutesFromChildren function has been modified to accept the breadcrumb route.
+
+// UTILS
+
+function invariant(cond: any, message: string): asserts cond {
+  if (!cond) throw new Error(message);
+}
+
+/**
+ * Creates a route config from a React "children" object, which is usually
+ * either a `<Route>` element or an array of them. Used internally by
+ * `<Routes>` to create a route config from its children.
+ *
+ * @see https://reactrouter.com/docs/en/v6/api#createroutesfromchildren
+ */
+export function createRoutesFromChildren(
```

Comment:
Is the idea here that users should use this method _instead_ of the one provided by `react-router`? My only concern there is that we'll have to be careful to keep it up to date whenever `react-router` is updated.

Reply:
That makes sense... Ideally `react-router` will merge your change so people can use their `createRoutesFromChildren`... in the meantime, I'm subscribed to all `react-router` updates, so if they happen to end up changing this method in a future update we can copy over their changes here.